src/AppBundle/Form/Settings/ProfileType.php

```diff
@@ -14,29 +14,38 @@
  */
 class ProfileType extends AbstractType
 {
-    /**
-     * @var User
-     */
-    private $loggedInUser = null;
+    protected $validationGroups;
 
     /**
-     * @param $loggedInUser
+     * ProfileType constructor.
+     * @param $validationGroups array
      */
-    public function __construct(User $loggedInUser)
+    public function __construct($validationGroups)
     {
-        $this->loggedInUser = $loggedInUser;
+        $this->validationGroups = $validationGroups;
     }
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $loggedInUser = $builder->getData();
```

Comment:
Q: Not sure why this is different from the loggedInUser that was injected into the class?